bin/dsh

```diff
@@ -26,6 +26,7 @@ DRUDE_COMMANDS_PATH=".drude/commands"
 # Network settings
 DRUDE_IP="192.168.10.10"
 DRUDE_SUBNET="192.168.10.1/24"
+DRUDE_SUBNET_RHEL="192.168.10.1"
```

Comment:
I don't like that we have two very similar constants.

Maybe do `DRUDE_SUBNET="192.168.10.1`, and append `/24` to it in `install_subnet_ubuntu`.